.github/labeler-issue.yml

```
@@ -0,0 +1,12 @@
+api:
+  - '(Devopness API)'
+website:
+  - '(Devopness website)'
+web-app:
+  - '(Devopness web app)'
+docs:
+  - '(Documentation)'
+sdk-js:
+  - '(SDK-JS)'
```

If we could keep issues and PRs labels in sync, that would make it easier to filter issues and PRs by common labels.
For reference, see: https://github.com/devopness/devopness/blob/main/.github/labeler-pr.yml#L16

```suggestion
sdk:
  - 'SDK'
sdk-javascript:
  - '(SDK-JS)'
```